### ISSUE TITLE
Cannot read Skills anymore

### ISSUE THREAD

**Issue by Svenja001** [stardew-access/stardew-access#423]
If I press e and go to the Skills Tab, I cannot read the Contents anymore, there are only Numbers displayed. This has been around since the new Stardew Update 1.6.9

**Comment by khanshoaib3** [comment]
@Svenja001 Do you happen to have spacecore installed?

**Comment by danielw97** [comment]
@khanshoaib3, another solution to this problem I've just found for those of us who have Space Core installed, there's a key in the config.json   "CustomSkillPage".
Setting it to false also fixes the problem for me, like this:

  "CustomSkillPage": false,

**Comment by Svenja001** [comment]
Yes I have SpaceCore, a Friend of mine also has the Issue so we didn't think about this being a Reason for breaking the Skills. @khanshoaib3 your Fix made it readable again so I guess this can be closed and referred in the Fixingcommit
